### ISSUE TITLE
Autocomplete touch devices issue

### ISSUE THREAD

**Issue by betocantu93** [miguelcobain/ember-paper#884]
Found a curious bug, while on touch devices (tested on Chrome devtools), if you click outside of the autocomplete after selecting an item from a dynamic list generated by a server query, the onSelectionChange method is executed again with a null item. This doesn't happen at all while on mouse devices, which makes this hard to debug. I also think this might be related to #767 and #787

**Comment by betocantu93** [comment]
Tracked it down to onInput, seems it gets triggered with the event "change".

**Comment by betocantu93** [comment]
Found the root, handleInputLocal method in paper-autocomplete-trigger, sends the action select with null if there is something already selected, it turns that on tocuh devices, handleInputLocal gets triggered by tapping outside of the trigger, which leads to the bug because it replaces the selected item with null.

My solution so far is to check for the type of the event, like this:

/components/paper-autocomplete-trigger.js

```
  handleInputLocal(e) {
      // If something is already selected when the user types, it should clear selection
      if (this.get('selected') && e.type === "input") {
        this.get('select').actions.select(null);
      }
      this.get('onInput')(e.target ? e : { target: { value: e } });
      this.set('text', e.target ? e.target.value : e);
    }
```

**Comment by betocantu93** [comment]
I guess this is solved by #956 . Closing for now.
